src/omnisharp/server.ts

```diff
@@ -515,8 +515,11 @@ export class OmniSharpServer {
 
     private async _doConnect(options: Options): Promise<void> {
 
-        this._serverProcess.stderr.on('data', (data: any) => {
-            this._fireEvent('stderr', String(data));
+        this._serverProcess.stderr.on('data', (data: Buffer) => {
```

Comment:
Shouldn't we add the same handling to stdio?

Reply:
Done. Thanks!